Assets/HoloToolkit/Input/Scripts/InputSources/EditorInputSource.cs

```diff
@@ -248,16 +251,18 @@ public override bool TryGetMenu(uint sourceId, out bool isPressed)
 
         private void Awake()
         {
-#if !UNITY_EDITOR
-            Destroy(gameObject);
-#else
+            if (Application.isEditor)
```

Comment:
This check is backwards. This is the editor input script, so we want it in the editor

Reply:
Good catch!